en/02_Developer_Guides/15_Customising_the_Admin_Interface/How_Tos/Customise_CMS_Pages_List.md

```diff
@@ -49,7 +49,7 @@ class NewsPage extends Page
 }
 ```
 
-We'll now add an `Extension` subclass to `LeftAndMain`, which is the main CMS controller.
+We'll now add an `Extension` subclass to `LeftAndMain`, which is the main CMS UI controller.
```

Comment:
`AdminController` is the main CMS controller, but `LeftAndMain` _is_ still the main CMS _UI_ controller which is what this section is talking about.